src/components/list-view/list-view.tsx

```diff
@@ -1,20 +1,86 @@
 import React, { PureComponent } from "react";
-import { View, Text } from "react-native";
+import { View, FlatList } from "react-native";
 import { styles } from "./style";
+import { EventCard } from "./event-card";
+import { data } from "../../../tst/mock-data";
```

Comment:
hmm good question. I can look into it

Reply:
actually ill add that in the DB change that was the DB change is 100% focused on database data + database data tests. either way were going to need mock-data just returned from the db instead of imported.